wurst/systems/trade/TradeShipSupervisor.wurst

```diff
@@ -75,11 +83,15 @@ public class TradeShipSupervisor
             return
 
         state = TradeShipSupervisorState.MOVING
+        this.orderListenerEnabled = false
         ship.issuePointOrder("move", nextStop.getPos())
+        this.orderListenerEnabled = true
         destroyCallback()
 
         callback = doPeriodically(1.0) cb ->
```

Comment:
Do we need this anymore? I think it's only to override commands issued by players, which this blocks now.

Reply:
I'd have the callback just to be safe

Reply:
OK. Add a comment to mark it for possible deprecation and I'll accept. I don't want to let it fall through the cracks at least.

Reply:
Done